config.sh

```diff
@@ -8,59 +8,83 @@ VERSION="1.0"
-	    deploy <imageName>				Binds everything together (app, db, container) through container group
-	    populate-db	<imageName>			Populates database with initial data
-	    delete <imageName>				Delete the group container and deletes created service if possible
-	    all <imageName>                 		Combines all necessary commands to deploy an app to IBM Cloud in a Docker container.
+	    install-tools                                   Installs necessary tools for config, like Cloud Foundry CLI
+    config-cli                                      Sets up and configures necessary CLI tools
+	    login                                           Logs into Bluemix and Container APIs
+    setup <clusterName>                             Sets up the clusters
+	    build                                           Builds Docker container from Dockerfile
+    run <imageName>                                 Runs Docker container, ensuring it was built properly
+	    stop <imageName>                                Stops Docker container, if running
+	    push-docker                                     Tags and pushes Docker container to IBM Cloud
+	    create-db <clusterName> <imageName>             Creates database service
+	    deploy                                          Binds everything together (app, db, container) through container group
+	    populate-db	<appURL> <username> <password>      Populates database with initial data
+	    delete <clusterName> <imageName>                Delete the created service and cluster if possible
+	    all <clusterName> <imageName>                   Combines all necessary commands to deploy an app to IBM Cloud in a Docker container.
 !!EOF
 }
 
 install-tools () {
-	brew tap cloudfoundry/tap
-	brew install cf-cli
-	cf install-plugin https://static-ice.ng.bluemix.net/ibm-containers-mac
+	curl -sL https://ibm.biz/idt-installer | bash
+    bx plugin install container-service -r Bluemix
+}
+
+config-cli () {
+    printf "Installing Kubernetes CLI...\n"
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/darwin/amd64/kubectl
+    chmod +x ./kubectl
+    sudo mv ./kubectl /usr/local/bin/kubectl
+    printf "\nInstalling IBM Cloud Container Service plugin\n"
+    bx plugin install container-registry -r Bluemix
+    bx plugin list
 }
 
 login () {
 	echo "Setting api and login tools."
-	cf api https://api.ng.bluemix.net
-	cf login
-	cf ic login
+	bx login -a $LOGIN_URL
+}
+
+setup () {
+    if [ -z "$1" ]
+    then
+        echo "Error: setup failed, cluster name not provided."
+    return
+    fi
+
+    bx cr login
+    bx cr namespace-add $NAME_SPACE
+    bx cs workers $1
+    bx cs cluster-config $1
+
+    MAGENTA='\033[1;35m'
+    NC='\033[0m'
+    printf "\n${MAGENTA}Copy and paste the command that is displayed in your terminal to set the KUBECONFIG environment variable${NC}\n"
```

Comment:
This part is confusing to me.  The script does not pause to allow me to export the KUBECONFIG, so as it stands the automatic `all` flow will not work correct?   I believe the **bx dev** cli does this automatically, though it does not persist.